packages/core-mobile/app/screens/network/NetworkDetails.tsx

```diff
@@ -44,6 +44,8 @@ export default function NetworkDetails({
 
   function connect(): void {
     dispatch(setActive(chainId))
+    goBack()
+    goBack()
```

Comment:
we could merge `NetworkDetailsScreen` and `NetworkDetails` into one, and call `navigation.navigate(navigate(AppNavigation.Portfolio.Portfolio)`.

but I'll approve as is, since we're moving toward the new design.